kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/batch/TableWriter.java

```diff
@@ -97,7 +97,7 @@ public void run() {
           currentIndex += currentBatchSize;
           successCount++;
         } catch (BigQueryException err) {
-          logger.warn("Could not write batch of size {} to BigQuery.", currentBatchList.size(), err);
+          logger.warn("Could not write batch of size {} to BigQuery. (Error code {})", currentBatchList.size(), err.getCode(), err);
```

Comment:
I wonder if it might also be beneficial to include `err.getError()` in the output here, since (like `error.getCode()`) that information is left out from just logging `err`.

What do you think about something like this?

```suggestion
          logger.warn("Could not write batch of size {} to BigQuery. Error code: {}, underlying error (if present): {}", currentBatchList.size(), err.getCode(), err.getError(), err);
```